src/content-tags-drawer/data/api.js

```diff
@@ -70,19 +70,14 @@ export async function getContentTaxonomyTagsCount(contentId) {
 }
 
 /**
- * Fetch meta data (eg: display_name) about the content object (unit/compoenent)
+ * Fetch meta data (eg: display_name) about the content object (unit/component)
  * @param {string} contentId The id of the content object (unit/component)
- * @returns {Promise<import("./types.js").ContentData | null>}
+ * @returns {Promise<import("./types.js").ContentData | undefined>}
  */
 export async function getContentData(contentId) {
   let url;
-  if (contentId.startsWith('lib-collection:')) {
-    // This type of usage_key is not used to obtain collections
-    // is only used in tagging.
-    return null;
-  }
 
-  if (contentId.startsWith('lb:')) {
+  if (contentId.startsWith('lb:') || contentId.startsWith('library:')) {
```

Comment:
Not sure what this change is for? `library:` isn't a valid key prefix, cf https://github.com/openedx/opaque-keys/blob/master/setup.py#L159-L189